README.md

```diff
@@ -432,3 +539,11 @@ It contains the following properties:
 - Does this work with the npm-replacements [yarn](https://github.com/yarnpkg/yarn), [pnpm](https://pnpm.js.org/), or [Bun](https://bun.sh/)?
 
   Yes! In all examples above, you may replace "`npm`" with "`yarn`", "`pnpm`", or "`bun`".
+
+```
```

Comment:
this and some others (Commented since `Generate "API" section` has been marked)

Reply:
It's caused by `ts-readme`, it will change as we'll use an other package.